app/components/app/app.component.ts

```diff
@@ -104,4 +114,14 @@ export class AppComponent {
       }
     });
   }
+
+  private translateConfig() {
+    var userLang = navigator.language;
+    userLang = /(de|en)/.test(userLang) ? userLang : 'en';
```

Comment:
Yes. The test look if the language from the device is german|english and if there is an other language, it will set it to english.